### PR TITLE
Add support for tox runner

### PR DESCRIPTION
**Pull request by cverna**
Signed-off-by: Clement Verna <cverna@tutanota.com>

### CODE REVIEW

**Comment by bstinsonmhk** [comment]
:+1:

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[tox]
+envlist = py27
+
+[testenv]
+deps = .[test]
+commands = python setup.py test
+passenv = HOME
```
Comment (bowlofeggs) [inline tox.ini:7]: Optional recommendation: Add diff-cover checking too, as described at https://fedora-infra-docs.readthedocs.io/en/latest/dev-guide/writing-tests.html#tox

**Review by bowlofeggs — APPROVED** [review]

**Comment by cverna** [comment]
@sayanchowdhury could you give a quick look at this PR ?

**Review by sayanchowdhury — APPROVED** [review]
:ballot_box_with_check:

**Pull request merged**